lib/assets/stylesheets/chef/components/icon.scss

```diff
@@ -5,17 +5,281 @@ name: icon
+
+  <!-- Then, reference each icon individually: -->
+  <svg class="icon_chef_delivery light-blue">
+    <use xlink:href="#icon_chef_delivery"></use>
+  </svg>
+</body>
 ```
+
+Lastly, you can use them with standard `img` tags:
+
+```html_example
+<img src="/assets/images/icons/icon_chef_delivery.svg" width="75" height="75">
+```
+
+Keep in mind, though, that using this method limits your ability to style the
+icons with CSS.
+
+### The Complete List
+
+```html_example
+<div id="all-icons">
+  <i class="icon_chef_analytics"></i>
+  <i class="icon_chef_client"></i>
+  <i class="icon_chef_community"></i>
+  <i class="icon_chef_delivery"></i>
+  <i class="icon_chef_dk"></i>
+  <i class="icon_chef_manage"></i>
+  <i class="icon_chef_ohai"></i>
+  <i class="icon_chef_push_jobs"></i>
+  <i class="icon_chef_server"></i>
+  <i class="icon_chef_supermarket"></i>
+  <i class="icon_chef_unikitty"></i>
+  <i class="icon_logos_centos"></i>
+  <i class="icon_logos_github"></i>
+  <i class="icon_logos_macosx"></i>
+  <i class="icon_logos_redhat"></i>
+  <i class="icon_logos_rubygems"></i>
+  <i class="icon_logos_ubuntu"></i>
+  <i class="icon_logos_windows"></i>
+  <i class="icon_node_types_application"></i>
+  <i class="icon_node_types_bare_metal"></i>
+  <i class="icon_node_types_container"></i>
+  <i class="icon_node_types_middleware"></i>
+  <i class="icon_node_types_network_device"></i>
+  <i class="icon_node_types_platform"></i>
+  <i class="icon_node_types_private_cloud"></i>
+  <i class="icon_node_types_public_cloud"></i>
+  <i class="icon_node_types_server"></i>
+  <i class="icon_node_types_storage_device"></i>
+  <i class="icon_node_types_virtual_machine"></i>
+  <i class="icon_node_types_virtual_platform"></i>
+  <i class="icon_objects_cookbook_attributes"></i>
+  <i class="icon_objects_cookbook_cookbooks"></i>
+  <i class="icon_objects_cookbook_definitions"></i>
+  <i class="icon_objects_cookbook_files"></i>
+  <i class="icon_objects_cookbook_libraries"></i>
+  <i class="icon_objects_cookbook_metadata"></i>
+  <i class="icon_objects_cookbook_providers"></i>
+  <i class="icon_objects_cookbook_recipes"></i>
+  <i class="icon_objects_cookbook_resources"></i>
+  <i class="icon_objects_cookbook_templates"></i>
+  <i class="icon_objects_cookbook_tests"></i>
+  <i class="icon_objects_cookbook_versions"></i>
+  <i class="icon_objects_server_api"></i>
+  <i class="icon_objects_server_client"></i>
+  <i class="icon_objects_server_cookbook"></i>
+  <i class="icon_objects_server_data_bag"></i>
+  <i class="icon_objects_server_environment"></i>
+  <i class="icon_objects_server_group"></i>
+  <i class="icon_objects_server_node"></i>
+  <i class="icon_objects_server_organization"></i>
+  <i class="icon_objects_server_policy"></i>
+  <i class="icon_objects_server_roles"></i>
+  <i class="icon_objects_server_run_list"></i>
+  <i class="icon_objects_server_search"></i>
+  <i class="icon_objects_server_user"></i>
+  <i class="icon_objects_server_workstation"></i>
```

Comment:
Would it be too much of a pain to put `title` attributes on these so you could hover over them to see what they mean?